src/orng.cpp

```diff
@@ -383,10 +503,57 @@ void orng::resetsuspen(const eosio::name &oracle)
     require_auth(get_self());
     oracles_table_type ot(get_self(), get_self().value);
     auto it = ot.require_find(oracle.value, "unknown oracle");
-    ot.modify(it, same_payer, [&](auto &r){ 
+    ot.modify(it, same_payer, [&](auto &r){
         r.strikes=0;
-        r.suspended=false; 
+        r.suspended=false;
     });
+
+    // Reactivate stipend accrual
+    ostip_table_type ostip_t(get_self(), get_self().value);
+    auto stip_itr = ostip_t.find(oracle.value);
+    if (stip_itr != ostip_t.end()) {
+        // Accrue stipend up to now before reactivating
+        auto now = current_time_point();
+        uint64_t accrued = _accrue_stipend(oracle, now);
```

Comment:
Do we need to update `accured`, at this point oracle still suspended and _accrue_stipend return 0

Reply:
I am torn about this logic too. On the one hand it is correct and GENERALLY we want the oracle to not be acruing any rewards when suspended (which is the case with the current logic). This logic leaves us the flexibility to change that AND shoot ourselves in the foot if we change something in code but do not intend to reward oracles while they are suspended. That's why I think we need the VRT tests so we can do proper integration tests to enforce those behaviours. As-is qtest is inadequate for that. I specifically made a test ticket for this case already. So I am fine with this so long as we address confirming that suspended oracles do not acrue rewards via an integration test in the near future